### PR TITLE
Feature/safe create table

### PR DESCRIPTION
**Pull request by SukitOwl**

### CODE REVIEW

File: mod_ngarn/utils.py [hunk 1]
```diff
@@ -64,12 +64,29 @@ def notify_channel(queue_table: str) -> str:
         raise ImportNotFoundException(e)
 
 
+async def is_table_exists(name: str) -> bool: 
+    cnx = await get_connection()
+    schema_query = "schemaname || '.' ||" if len(name.split('.')) == 2 else ""
```
Comment (tleekkul) [inline mod_ngarn/utils.py]: We can abstract this to be a util function

File: mod_ngarn/utils.py [hunk 2]
```diff
@@ -64,72 +68,104 @@ def notify_channel(queue_table: str) -> str:
...
         )
+    )
+
+
+async def create_table(name: str):
+    print(f"Creating table {name}...")
+    cnx = await get_connection()
+    async with cnx.transaction():
+        if await is_table_exists(name):
+            print("Table {queue_table} alrady exsits skipped".format(queue_table=name))
```
Comment (tleekkul) [inline mod_ngarn/utils.py]: exsits -> exists

Comment (KayEss) [reply mod_ngarn/utils.py]: alrady -> already

Comment (KayEss) [reply mod_ngarn/utils.py]: "Table {queue_table} already exists, skipped"

Note extra comma :-)

File: mod_ngarn/utils.py [hunk 3]
```diff
@@ -20,7 +21,10 @@ class ModuleNotfoundException(Exception):
 
 
 def sql_table_name(queue_table: str) -> str:
-    return (".").join([f'"{x}"' for x in queue_table.replace('"', "").split(".")])
+    quote_table_name = [f'"{x}"' for x in queue_table.replace('"', "").split(".")]
+    if len(quote_table_name) == 1:
+        quote_table_name.insert(0, '"public"')
```
Comment (tleekkul) [inline mod_ngarn/utils.py]: qtbl = ['"public"'] + quote_table_name

**Pull request merged**